classes/03class/exercises/c03-docker01/gepifanio/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
+ENV TERRAFORM_VERSION=0.12.20
+CMD terraform -v
+RUN apk update && \
+    wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/bin
```

Comment:
Although it works because of the `CMD` behaviour to execute only when the container runs.
`CMD` instructions are commonly put at the end of Dockerfile.